content/app/development/ux/components/alertComponent/_index.en.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Alert
+description: This component displays an important message to the user. The component can be used to display a message with different levels of severity.
+hidden: false
+---
+
+## Usage
+
+Alerts are commonly used to display important information to users that should be highlighted.
+
+* [Page layout JSON schema](https://altinncdn.no/schemas/json/layout/layout.schema.v1.json)
+
+## Component properties
+
+| **Property** | **Title**    | **Type**                                       | **Description**                                                                                |
+|--------------|--------------|------------------------------------------------|------------------------------------------------------------------------------------------------|
+| `severity`   | Severity     | `"success" \| "info" \| "danger" \| "warning"` | String value indicating the severity level of the alert. This affect the styling of the alert. |
+
+### Properties Description
+
+The component has these properties:
+
+#### severity
+
+The severity of the alert. This affects the styling of the alert.
+
+
+#### textResourceBindings
+
+`title` and `description` can be configured with `textResourceBindings` to display text from the resource file.
+
+## Accessibility
+
+The component does not get `role="alert"` by default. This means that screen readers will not treat this as an alert.
+An `Alert` should only have `role="alert"` if it appears in the ui as a result of an action the user has taken.
+We automatically add `role="alert"` if the `Alert` has a `hidden` property set to `false`. This means that the `Alert` 
+became visible to the user based on an action the user has taken.
```

Comment:
If i was an app developer, that wouldn't make any sense to me. You're talking about an internal representation in app-frontend-react (i.e. the result of an evaluated expression). So I suggest saying that instead; 'role=alert will be added if you use an expression in the `hidden` property' or something of that effect.

Reply:
I agree. Will change the wording!